Requesting signoff for Mold Subsystem

docrobbins4 opened the pull request.

jacksonrwoodard committed:
Create dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

JesseTNRoberts commented on Documentation/Signoffs/dylanrobbins-mold-signoff.md:
```diff
@@ -0,0 +1,65 @@
+|  1  | Shall be able to detect temperature ranges between 55&deg; F - 85&deg; F and humidity levels exceeding 50% RH. | Project Team |
+|  2  | Shall be able to communicate through I2C protocol and send data every hour to its local ESP32-H2 transmitter. | Project Team |
+|  3  | Shall be protected and not exposed to harsh environmental conditions to prevent any damage to the sensor. | All External Stakeholders, Ethics, & Team Supervisor |
+|  4  | Shall give precise readings to properly evaluate and determine if mold like conditions are present. | Project Team |
```
this is not specific enough.

JesseTNRoberts replied:
I have addressed that this is not sufficiently precise.

docrobbins4 replied:
I added specific reading ranges to be more precise in the constraint.

JesseTNRoberts reviewed (commented):
Nothing here about sleep mode. I'm not confident that these are wired correctly to permit communication.

JesseTNRoberts reviewed (commented):
Prove that the connections made will function based on datasheets.

docrobbins4 committed:
updated constraint 4 to be more precise

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 committed:
Update dylanrobbins-mold-signoff.md

docrobbins4 commented:
> Nothing here about sleep mode. I'm not confident that these are wired correctly to permit communication.

The sleep mode is targeted towards the ESP32-H2 on the communication side. I added on to the constraint to be more specific about the sleep mode. I added a reference to the datasheet in the schematic section stating that the ESP32-H2 can support I2C on any GPIO pin.

docrobbins4 commented:
> Prove that the connections made will function based on datasheets.

I added a reference to the datasheet in the schematic section stating that the ESP32-H2 can support I2C on any GPIO pin.

JesseTNRoberts reviewed (commented):
Wireless has nothing to do with I2c

docrobbins4 committed:
took out the reference to wireless communication
